pkgs/development/python-modules/sjcl/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pycrypto
+}:
+
+buildPythonPackage rec {
+  pname = "sjcl";
+  version = "0.2.1";
+
+  format = "pyproject";
```

```suggestion
  format = "setuptools";
```

I can't find a `pyproject.toml` file in the repo or sdist.

---

changed with [e119b23](https://github.com/NixOS/nixpkgs/pull/167951/commits/e119b230877703a025cd50aa9b317f686ab46dd6)